shadowapi/src/main/java/org/robolectric/util/ReflectionHelpers.java

```diff
@@ -167,7 +169,7 @@ public Void run(Class<?> traversalClass) throws Exception {
    */
   public static void setField(Class<?> type, final Object object, final String fieldName, final Object fieldNewValue) {
     try {
-      Field field = getDeclaredField(type, fieldName);
+      Field field = type.getDeclaredField(fieldName);
```

Comment:
What about `type` is null?

Reply:
To some extent this is just a revert of 4af252cce5f6d06f251e230d2c86becbdbddcb1c, which caused a scary warning when running tests. There has never been a null check for type here.